### PR TITLE
Adding Grandine to various pages

### PR DESCRIPTION
**Pull request by arredr2**
## Description

I updated the following pages by adding Grandine to any mention of CL clients.
    - Nodes and clients (https://ethereum.org/en/developers/docs/nodes-and-clients)
    - Client diversity (https://ethereum.org/en/developers/docs/nodes-and-clients/client-diversity/#consensus-clients)
    - Ethereum Support (https://ethereum.org/en/community/support/#consensus-clients)
    - Adding staking products or services (https://ethereum.org/en/contributing/adding-staking-products/#node-or-client-tooling)
    - Ethereum Governance (https://ethereum.org/en/governance/#who-is-involved)

### CODE REVIEW

File: public/content/developers/docs/nodes-and-clients/index.md [hunk 1]
```diff
@@ -195,12 +195,19 @@ There are multiple consensus clients (previously known as 'Eth2' clients) to sup
 
 | Client                                                        | Language   | Operating systems     | Networks                                                          |
 | ------------------------------------------------------------- | ---------- | --------------------- | ----------------------------------------------------------------- |
+| [Grandine](https://docs.grandine.io/)                         | Rust       | Linux, Windows, macOS | Beacon Chain, Goerli, Sepolia, and more                           |
```
Comment (wackerow) [inline public/content/developers/docs/nodes-and-clients/index.md:198]: Please move this to last and mark with `(beta)` at the end given we are not yet at a v1 production release.

File: public/content/developers/docs/nodes-and-clients/index.md [hunk 2]
```diff
@@ -195,12 +195,19 @@ There are multiple consensus clients (previously known as 'Eth2' clients) to sup
...
 | [Nimbus](https://nimbus.team/)                                | Nim        | Linux, Windows, macOS | Beacon Chain, Goerli, Sepolia, Ropsten, and more                  |
 | [Prysm](https://docs.prylabs.network/docs/getting-started/)   | Go         | Linux, Windows, macOS | Beacon Chain, Gnosis, Goerli, Pyrmont, Sepolia, Ropsten, and more |
 | [Teku](https://consensys.net/knowledge-base/ethereum-2/teku/) | Java       | Linux, Windows, macOS | Beacon Chain, Gnosis, Goerli, Sepolia, Ropsten, and more          |
 
+### Grandine {#grandine}
+
+Grandine is the latest consensus client implementation, written in Rust under the GPL-3.0 license. It is maintained by the Grandine Core Team and is fast, high-performance and lightweight. It fits a wide range of stakers from solo stakers running on low-resource devices such as Raspberry Pi to large institutional stakers running tens of thousands of validators.
+
+Documentation can be found in the [Grandine Book](https://docs.grandine.io/)
+
```
Comment (wackerow) [inline public/content/developers/docs/nodes-and-clients/index.md:210]: Please move to end (below Teku)

File: public/content/developers/docs/nodes-and-clients/client-diversity/index.md [hunk 3]
```diff
@@ -80,6 +80,8 @@ Addressing client diversity requires more than individual users to choose minori
 
 [Prysm](https://docs.prylabs.network/docs/getting-started)
 
+[Grandine](https://github.com/grandinetech/grandine)
+
```
Comment (wackerow) [inline public/content/developers/docs/nodes-and-clients/client-diversity/index.md:84]: Would avoid this listing until Grandine is out of beta with at least a v1 release

```suggestion
```

File: public/content/developers/docs/nodes-and-clients/index.md (hunk 2 above)
Comment (wackerow) [reply public/content/developers/docs/nodes-and-clients/index.md:210]: Would avoid phrasing such as "the latest consensus client" as this can change.

**Review by wackerow — CHANGES_REQUESTED** [review]
Thanks @arredr2! Correct me if I'm wrong, but Grandine appears to still be in beta, with release version <v1 still. Left some comments; don't see why we can't still list Grandine, but in the past we've marked those not yet fully in production as `(beta)` and graduate it over time when ready.

**Comment by arredr2** [comment]
@wackerow thanks for your comments! I have updated based on your suggestions.

**Pull request merged**